PersonStatus[] people
int8 people_tracked
int8 people_interested
